src/tooltip/index.jsx

```diff
@@ -0,0 +1,68 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+class Tooltip extends React.Component {
+  constructor (props) {
+    super(props);
+    this.state = {
+      showTooltip: false
+    };
+    this.handleRenderTooltip = this.handleRenderTooltip.bind(this);
+    this.handleCalculateTooltipPosition = this.handleCalculateTooltipPosition.bind(this);
+  }
+
+  shouldComponentUpdate (nextProps, nextState) {
+    const { showTooltip } = this.state;
+    if (nextState.showTooltip === showTooltip) {
+      return false;
+    }
+    return true;
+  }
+
+  handleRenderTooltip () {
+    const { showTooltip } = this.state;
+    this.setState({ showTooltip: !showTooltip });
+  }
+
+  handleCalculateTooltipPosition (event) {
+    const tooltipClassName = 'tooltip-wrapper';
+    let tooltip = document.getElementsByClassName(tooltipClassName);
+    const xPositionOffset = 25;
+    const yPositionOffset = 65;
+
+    tooltip[0].style.left = event.clientX - xPositionOffset + 'px';
+    tooltip[0].style.top = event.clientY - yPositionOffset + 'px';
+  }
+
+  render () {
+    const { children, tooltipText, isTooltipActive } = this.props;
```

Comment:
is isTooltipActive needed when pointer-events have been disabled?

Reply:
nope it's not needed